CONTRIBUTING.md

```diff
@@ -0,0 +1,48 @@
+# Contribution Guidelines
+
+Thanks for your interest in contributing to PyCon India Blog. Your valuable contribution is much appreciated.
+
+The following is a set of guidelines for contributing to PyCon India Blog.
+
+## How Can I Help?
+
+Start looking through the issues and watch out for any issues with the `bug` or `help wanted` labels. Issues marked with these labels haven't been assigned to anyone yet and we would love for you to work on them.
+
+## Request a feature
+
+If you think there is a cool feature you would like to add please create an issue for that and we will assign that to you if the feature looks good to add.
+
+## Reporting bugs
+
+If you find any bugs, it would be great if you would let us know so we can look into it. You can do so by creating an issue. If you don't know how to create an issue, please follow [these instructions](https://help.github.com/articles/creating-an-issue/). Screenshots and notes for how you encountered the bug are really helpful when reporting bugs!
+
+## Creating Pull Requests
+
+After you've created a patch that fixes an issue, please submit a pull request for it. Mention **@pythonindia/pycon-2018-team** to get the Pull Request reviewed. If you're not sure how to create a pull request, please follow [these instructions](https://help.github.com/articles/creating-a-pull-request/). In general we'd like to ask you to please target the master branch with your pull requests.
+
+We recommend creating a new branch for each feature task/bug since it makes easier to edit commits in future if needed.
+
+## Reviewing Issues and Pull Requests
+
+You're welcome to review issues and pull requests, provide feedback, and answer questions. Please remember to be friendly and respectful when commenting on issues or pull requests, and always follow our [Code of Conduct](https://in.pycon.org/2018/coc.html). It's okay for people to make mistakes and ask questions! 
+
+
+## How Can I Get Help?
+
+If you have a question or need help, please leave a comment on the issue you have a question about or are working on. We will try to answer your question and help you as soon as possible.
+
+If you are stuck or have some questions on an issue that you are working on, then feel free to ask the community on the mailing list of PyCon India. Be sure to mention the Issue # and mention **Website Contribution** in Subject line so it will be easier for people to help you in this regard.
+
+## Workflow & Merging Pull Requests
+<!--
+- We will be using master branch for development and gh-pages branch for viewing the website directly using github pages
+-->
+- Reviews to Pull Request created can be requested by mentioning **@pythonindia/pycon-2018-team** in the Pull Request body or comments
```

Comment:
Again, no need to mention specific teams.

Reply:
updated